about/about-serverless.adoc

```diff
@@ -22,7 +23,7 @@ In addition, the {ServerlessProductName} documentation is also available on the
 For additional information about the {ServerlessProductName} life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift#ossrvless[Platform Life Cycle Policy].
 ====
 
-// add something about CLI tools?
+// Add something about CLI tools?
```

Comment:
Probably don't need the two changes in this file; it would cut down on the churn by merging only one file.

Reply:
I made the changes here in this .adoc file deliberately because my netlify preview was not generating even if I made the changes in the `_topic_map.yml` file. And it worked. So this is sort of a workaround my DPM (Brian) suggested. Thank you so much.